internal/constants/constants.go

```diff
@@ -122,8 +125,16 @@ const (
 	CosignPublicKey = `-----BEGIN PUBLIC KEY-----
 MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEf8F1hpmwE+YCFXzjGtaQcrL6XZVT
 JmEe5iSLvG1SyQSAew7WdMKF6o9t8e2TFuCkzlOhhlws2OHWbiFZnFWCFw==
------END PUBLIC KEY-----`
+-----END PUBLIC KEY-----
+`
 )
 
 // VersionInfo is the version of a binary. Left as a separate variable to allow override during build.
 var VersionInfo = "0.0.0"
+
+// CosignPublicKeyBase64 converts CosignPublicKey to base64.
+func CosignPublicKeyBase64() string {
```

Comment:
Mostly because I was confused from having a `constants_test.go` and having this function almost looks like having 2 sources of truth, I'd propose to always use `CosignPublicKey` and convert it where needed. I.e. doing the conversion in `VerifySignature(...)`.